triton/README.md

```diff
@@ -69,26 +69,27 @@ cp <bpe_model_path> <jit_model_dir>
 
 Streaming Model Export:
 ```bash
-cp $SHERPA/triton/scripts/*onnx*.py $ICEFALL_DIR/egs/librispeech/ASR/pruned_stateless_transducer3/
+cp $SHERPA/triton/scripts/*onnx*.py $ICEFALL_DIR/egs/wenetspeech/ASR/pruned_stateless_transducer5/
```

Comment:
Looks like you are using `SHERPA_SRC` before but here you are using `SHERPA`.

Reply:
Thanks, fixed it.